test/main.js

```diff
@@ -1,15 +1,24 @@
 'use strict';
 
 var assert = require('chai').assert;
-var cheerio = require('cheerio');
-var HTMLUglify = require('../lib/main.js');
 
+var parser = require('posthtml-parser');
+var render = require('posthtml-render');
+var walk = require('posthtml/lib/api').walk;
+
+var plugin = require('../lib/main');
+var HTMLUglify = plugin.HTMLUglify;
 var htmlUglify = new HTMLUglify();
 
+function parse(html) {
+  var tree = parser(html);
+  tree.walk = walk;
```

Comment:
This is just adding the `walk` function to the tree returned by parser, like happens here: https://github.com/posthtml/posthtml/blob/master/lib/posthtml.js#L141